Site/admin/SiteCommentVisibilityCellRenderer.php

```diff
@@ -1,52 +1,41 @@
 <?php
 
 /**
- * A cell renderer for rendering visibility of site comments
+ * A cell renderer for rendering visibility of site comments.
  *
- * @package   Site
  * @copyright 2008-2016 silverorange
  * @license   http://www.gnu.org/copyleft/lesser.html LGPL License 2.1
  */
 class SiteCommentVisibilityCellRenderer extends SwatCellRenderer
 {
-	// {{{ public properties
-
-	public $status;
-	public $spam;
-
-	// }}}
-	// {{{ public function render
-
-	public function render()
-	{
-		if ($this->spam) {
-			$color = 'Red';
-			$title = Site::_('Spam');
-
-		} else {
-			$title = SiteComment::getStatusTitle($this->status);
-
-			switch ($this->status) {
-			case (SiteComment::STATUS_UNPUBLISHED):
-				$color = 'Red';
-				break;
-
-			case (SiteComment::STATUS_PUBLISHED):
-				$color = 'Green';
-				break;
-
-			case (SiteComment::STATUS_PENDING):
-				$color = 'Yellow';
-				break;
-			}
-		}
-
-		// TODO: output the title in the color, or use some sort of visual
-		// representation for the statuses, like a traffic light
-		echo $title;
-	}
-
-	// }}}
+    public $status;
+    public $spam;
+
+    public function render()
+    {
+        if ($this->spam) {
+            $color = 'Red';
+            $title = Site::_('Spam');
+        } else {
+            $title = SiteComment::getStatusTitle($this->status);
+
+            switch ($this->status) {
```

Comment:
This could be replaced with match, or deleted as the var is unused with a TODO

Reply:
Refactored and simplified.